tests/io/v0_0_0/test_write.py

```diff
@@ -117,3 +118,64 @@ def test_write_collection(self):
 
                     self.assertEqual(tiles[0].numpy_array.all(), expected.all())
                     self.assertIsNotNone(tiles[0].sha256)
+
+    def test_checksum_on_write(self):
+        """
+        Generate a tileset consisting of a single TIFF tile.  Load it and then write it back out
```

Comment:
You're writing it as a numpy tile _because this will change the checksum_, right? If so, please make that explicit in this docstring.

Reply:
Yes.